notebooks/official/pipelines/lightweight_functions_component_io_kfp.ipynb

```diff
@@ -1,874 +1,771 @@
+   "execution_count": null,
+   "metadata": {
+    "id": "set_service_account"
+   },
+   "outputs": [],
+   "source": [
+    "SERVICE_ACCOUNT = \"[your-service-account]\"  # @param {type:\"string\"}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "autoset_service_account"
+   },
+   "outputs": [],
+   "source": [
+    "import sys\n",
+    "\n",
+    "IS_COLAB = \"google.colab\" in sys.modules\n",
+    "if (\n",
+    "    SERVICE_ACCOUNT == \"\"\n",
+    "    or SERVICE_ACCOUNT is None\n",
+    "    or SERVICE_ACCOUNT == \"[your-service-account]\"\n",
+    "):\n",
+    "    # Get your service account from gcloud\n",
+    "    if not IS_COLAB:\n",
+    "        shell_output = !gcloud auth list 2>/dev/null\n",
+    "        SERVICE_ACCOUNT = shell_output[2].replace(\"*\", \"\").strip()\n",
+    "\n",
+    "    if IS_COLAB:\n",
+    "        shell_output = ! gcloud projects describe  $PROJECT_ID\n",
+    "        project_number = shell_output[-1].split(\":\")[1].strip().replace(\"'\", \"\")\n",
+    "        SERVICE_ACCOUNT = f\"{project_number}-compute@developer.gserviceaccount.com\"\n",
+    "\n",
+    "    print(\"Service Account:\", SERVICE_ACCOUNT)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "set_service_account:pipelines"
+   },
+   "source": [
+    "#### Set service account access for Vertex AI Pipelines\n",
+    "\n",
+    "Run the following commands to grant your service account access to read and write pipeline artifacts in the bucket that you created in the previous step -- you only need to run these once per service account."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "set_service_account:pipelines"
+   },
+   "outputs": [],
+   "source": [
+    "! gsutil iam ch serviceAccount:{SERVICE_ACCOUNT}:roles/storage.objectCreator $BUCKET_URI\n",
+    "\n",
+    "! gsutil iam ch serviceAccount:{SERVICE_ACCOUNT}:roles/storage.objectViewer $BUCKET_URI"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "setup_vars"
+   },
+   "source": [
+    "### Set up variables\n",
+    "\n",
+    "Next, set up some variables used throughout the tutorial.\n",
+    "### Import libraries and define constants"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "import_aip:mbsdk"
+   },
+   "outputs": [],
+   "source": [
+    "from google.cloud import aiplatform \n",
+    "from typing import NamedTuple\n",
+    "\n",
+    "import kfp\n",
+    "from kfp import compiler, dsl\n",
+    "from kfp.dsl import (Artifact, Dataset, Input, InputPath, Model, Output,\n",
+    "                     OutputPath, component)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "pipeline_constants"
+   },
+   "source": [
+    "#### Vertex AI Pipelines constants\n",
+    "\n",
+    "Setup up the following constants for Vertex AI Pipelines:"
```

Comment:
Change: 

- Setup up the following constants for Vertex AI Pipelines

To:

- Set up the following constants for Vertex AI Pipelines